test/extended/quota/resourcequota.go

```diff
@@ -85,6 +85,74 @@ var _ = g.Describe("[sig-api-machinery][Feature:ResourceQuota]", func() {
 			})
 			o.Expect(err).NotTo(o.HaveOccurred())
 		})
+
+		g.It("The number of created persistent volume claims can not exceed the limitation", func() {
```

Comment:
Nit: the name will be concatenated with both the `g.Describe` above, so it'll be:
```
"[sig-api-machinery][Feature:ResourceQuota] Object count The number of created persistent volume claims can not exceed the limitation [Suite:openshift/conformance/parallel]"
```
so to make it look good, we need to:
1. make sure we start with lowercase
2. make the sentence "flow" nicely together.

In this case I'd suggest something similar to the other `g.It` in this file, so `should properly count the number of persistentvolumeclaims resources`. But that's something that doesn't have to be fixed in this PR.